src/table.js

```diff
@@ -679,3 +705,25 @@ export const findCellRectClosestToPos = $pos => {
     return map.rectBetween(cellPos, cellPos);
   }
 };
+
+// :: (rowIndex: number, predicate: (node: ProseMirrorNode, tr: Transaction, currentRowIndex: number) → boolean) → (tr: Transaction) → [?{pos: number, start: number, node: ProseMirrorNode}]
+// Returns a list of cells that match the provided predicate.
+//
+// ```javascript
+// filterPreviousCells(1, cell => cell.node.attrs.rowspan === 1)
+// ```
+export const filterPreviousCells = (rowIndex, predicate) => tr => {
```

Comment:
maybe its better to leave it as a private method for now, its a very specific low level operation that I don't expect people to use often

Reply:
maybe its better to also rename it to something like `filterCellsInRow`